workflows/mtv-plan/mtv.sw.yaml

```diff
@@ -84,6 +84,8 @@ states:
         nextState: GetPlan
   - name: PlanFailure
     type: operation
+    metadata:
+      errorMessage: '"MTV plan " + .migrationName + " is not ready for migration."'
```

Comment:
let's decide on same convention for error messages: either all ends with a dot or none. ATM it is a mix.

Reply:
I am not opinionated on so I will consider that every sentence has to end with a period and I'll unify all messages accordingly

Reply:
I did the opposite for a simple reason: some message end with a variable so I do not want to add another concat just to add a period